defaults.yaml

```diff
@@ -0,0 +1,11 @@
+---
+dbUser: 'probodb'
+dbPassword: 'password'
+dbName: 'probodb'
+dbType: 'postgres'
+dbHost: 'localhost'
+eventGroup: 'proboDb'
+eventTopic: 'build_events'
+eventVersion: 1
+pluginDirectory: null
+knex: {}
```

Comment:
maybe `nodeEnv: production` to make the [executable configurable](https://github.com/ProboCI/probo-db/pull/1/files#diff-ea4f1a242219c0bdf45c119a73a8fa20R16)